emukit/test_functions/quadrature_functions.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Test functions need some changing. At the very least our loop deals with UserFunction objects, so that's what test functions should provide.

Reply:
Yes, I'll leave that to when I do the tutorial notebook. I can remove these from the PR for now if that is better.